source/Cosmos.System2/Graphics/Canvas.cs

```diff
@@ -971,15 +969,15 @@ public virtual void DrawImage(Image image, int x, int y, int w, int h)
         /// <exception cref="ArgumentOutOfRangeException">Thrown on fatal error.</exception>
         public void DrawImageAlpha(Image image, int x, int y)
         {
-            var _pen = new Pen(Color.White);
+            var _color = Color.White;
```

Comment:
Unnecessary assignment to `_color`; the value of `Color.White` will always be replaced with `(Color.FromArgb(image.rawData[_x + _y * image.Width]));`